frame/scheduler/src/benchmarking.rs

```diff
@@ -160,6 +160,10 @@ benchmarks! {
 
 	// `service_task` when the task is a non-periodic, non-named, fetched call (with a known
 	// preimage length) and which is not dispatched (e.g. due to being overweight).
+	#[pov_mode = MaxEncodedLen {
+		// Use measured PoV size for the Preimages since we pass in a length witness.
+		Preimage::PreimageFor: Measured
+	}]
```

Comment:
> I don't see any code changes in preimages pallet to annotate that it should use empirical PoV size rather than the MEL-derived value. Am I missing something or is it not yet done (as I wrote above I don't think we should be merging until it's sorted).

It is done on the benchmark in case that the same storage item is accessed with and sometimes without length witness.  
Same for the whitelist pallet. @gavofyork